planetarium/NineChronicles
# Make color tweener and improve existing tweener

### Description

1. Make Color Tweener.
2. Refactor existing tweener that implement `DOTWeenBase`.
3. Change base class of some tweener components to `DOTweenBase`.

### How to test

1. Add components that implements `DOTweenBase` to UI.
2. Check `useCustomEaseCurve` field.
3. Try to use `GraphicColorTweener` component.

### Related Links

[기존 DOTweenBase 기반 트위너 개선](https://app.asana.com/0/958521740385861/1201259381741789/f)
[컬러 트위너 구현](https://app.asana.com/0/958521740385861/1201200786778969/f)
[트위너 정리](https://www.notion.so/planetarium/8c7ffa6c7ed44a27a5176bce26848282)

### Screenshot
![image](https://user-images.githubusercontent.com/48484989/139188618-d945f628-a09e-44d4-9c04-5d90a7d85e81.png)
![image](https://user-images.githubusercontent.com/48484989/139188641-f80549dc-53da-4a5d-960b-02577633f2ae.png)
![GIF 2021-10-28 오후 1-51-19](https://user-images.githubusercontent.com/48484989/139189242-7c5b478a-7a71-442e-912a-7bce457e9ab4.gif)
